adding Manage section and articles to docs (#558)
sharetribe/flex-docs: src/docs/concepts-management/manage-transactions/index.md

- Stock reservation quantity: The amount of stock reserved in this transaction.
- Transaction ID: Unique ID of the transaction in the Flex database.
- Transaction process: You can have multiple [transaction processes](https://www.sharetribe.com/docs/concepts/transaction-process/) active in your marketplace and each transaction process can have multiple versions. This field specifies which process and which version the transaction uses. 
- Customer paid: This field shows how much money the customer paid. It also includes a breakdown of the line items and the fee that your marketplace collects from the buyer.
- Provider received: This field shows how much money the provider will receive for this transaction. It also includes a breakdown of the line items charged and the fee that your marketplace collects from the seller. 
- Marketplace receives: This field shows the total of the customer and provider fees to determine the total marketplace commission on the transaction. Stripe’s fees are not separated and can only be viewed in your Stripe dashboard. 

### Timeline

> In the timeline, you see all the transitions that have been triggered so far in the transaction as well as the time when they took place and [the actor who initiated each transition](https://www.sharetribe.com/docs/operator-guides/transaction-process/#transitions). You also see the possible next transitions. If the actor of the next possible transition is the Operator (you), you will be able to initiate that transition in the timeline section. 

Reviewer: ```suggestion
In the timeline, you see all the transitions that have been triggered so far in the transaction as well as the time when they took place and [the actor who initiated each transition](https://www.sharetribe.com/docs/concepts/transaction-process/#transitions). You also see the possible next transitions. If the actor of the next possible transition is the Operator (you), you will be able to initiate that transition in the timeline section. 
```

### Activity 

> [All customer interactions in your marketplace](https://www.sharetribe.com/docs/operator-guides/transaction-process/#users-interact-through-transactions) are considered transactions in Flex. In the activity section, you will see the history of the transaction. This includes all the transitions that have occurred already, similar to the Timeline section, interwoven with the communication that has happened within the transaction. 

Reviewer: ```suggestion
[All customer interactions in your marketplace](https://www.sharetribe.com/docs/concepts/transaction-process/#users-interact-through-transactions) are considered transactions in Flex. In the activity section, you will see the history of the transaction. This includes all the transitions that have occurred already, similar to the Timeline section, interwoven with the communication that has happened within the transaction. 
```

The communication includes all the [notifications that have been sent](https://www.sharetribe.com/docs/references/transaction-process-format/#notifications) as a part of the transaction process, the written messages between users, and the reviews given at the end of the transaction, if available. 

### Extended data

> There are two separate sections for each type of extended data supported by transactions: protected data and metadata. You can read [an introduction to Extended data](https://www.sharetribe.com/docs/operator-guides/extended-data-introduction/) to learn more about different types of Extended Data. Unlike with [user cards](https://www.sharetribe.com/docs/concepts/console-manage-users/) and [listing cards](https://www.sharetribe.com/docs/concepts/console-manage-listings/), transaction public extended data cannot be edited directly through Console. You can change existing field values or even add new fields for the metadata, but, since protected data is captured at specific points of the transaction, it is seared in the transaction information. Transaction related protected data cannot be changed through Console. 

Reviewer: ```suggestion
There are two separate sections for each type of extended data supported by transactions: protected data and metadata. You can read [an introduction to Extended data](https://www.sharetribe.com/docs/concepts/extended-data-introduction/) to learn more about different types of Extended Data. Unlike with [user cards](https://www.sharetribe.com/docs/concepts/console-manage-users/) and [listing cards](https://www.sharetribe.com/docs/concepts/console-manage-listings/), transaction public extended data cannot be edited directly through Console. You can change existing field values or even add new fields for the metadata, but, since protected data is captured at specific points of the transaction, it is seared in the transaction information. Transaction related protected data cannot be changed through Console.
```

> At the bottom of the transaction card, you find the Events section. It contains a Flex CLI command that you can use to view all the events related to this transaction. Read more about [viewing events with Flex CLI](https://www.sharetribe.com/docs/flex-cli/view-events-with-flex-cli/) or visit the [Events reference in our Docs site](https://www.sharetribe.com/docs/references/events/). 

Reviewer: ```suggestion
At the bottom of the transaction card, you find the Events section. It contains a Flex CLI command that you can use to view all the events related to this transaction. Read more about [viewing events with Flex CLI](https://www.sharetribe.com/docs/how-to/view-events-with-flex-cli/) or visit the [Events reference in our Docs site](https://www.sharetribe.com/docs/references/events/). 
```
